pkg/fission-cli/cmd/function/test.go

```diff
@@ -107,14 +112,26 @@ func (opts *TestSubCommand) do(input cli.Input) error {
 		functionUrl.RawQuery = query.Encode()
 	}
 
-	var ctx context.Context
+	var (
+		ctx        context.Context
+		reqTimeout time.Duration
+	)
+
+	fnTestTimeout := input.Duration(flagkey.FnTestTimeout)
+	fnSpecTimeout := time.Duration(function.Spec.FunctionTimeout)
+
+	if input.IsSet(flagkey.FnTestTimeout) && (fnTestTimeout < fnSpecTimeout) {
+		reqTimeout = fnTestTimeout
+		console.Warn("timeout specified using flag is less than the actual functionTimeout specified for function")
```

Comment:
Add info what value we are setting.

`timeout specific is less than functionTimeout %s. Overriding value to %s`

Reply:
done